### DISCUSSION TITLE
Larger than memory cache

### DISCUSSION THREAD

**Discussion by salarali** [microsoft/garnet#847] [Q&A]
I am confused on how to achieve this? Would it possible if someone can guide me towards settings that accomplish this?
What I have is a 128GB memory and 500GB disk. All the data that exceeds 128 should go to the disk. Its unclear to me how I achieve this?

**Comment by donnyv** [comment]
I'm in the same boat. Picking up scrapes of info from different web sites. I wish the Garnet site had more info about how it stores, when, etc.
What I could gather there is a setting when configuring Garnet to use something called AOF. You can tell it to save at certain points in time. 
https://github.com/microsoft/garnet/discussions/166#discussioncomment-9243435

**Reply by donnyv** [reply]
This explains what happens when data doesn't fit in memory.
https://microsoft.github.io/garnet/docs/getting-started/compaction

**Comment by badrishc** [comment]
> What I have is a 128GB memory and 500GB disk

You have to configure index size, log memory size based on your total memory. Garnet will spill to disk beyond that size. The disk log will keep growing, and you can enable compaction to limit disk use. All the details are there in our docs but they may be scattered across various places. Feel free to contribute a page that describes everything in one place for this specific scenario.

**Reply by salarali** [reply]
Do I have to use aof like the other comment states?
